README.md

```diff
@@ -122,6 +122,22 @@ for installing cookbooks defined within the manifest. This allows for easy trans
 and direct installation of a manifest without the requirement of re-pulling information
 from sources.
 
+### Infrastructure manifests
+
+Batali aims to solve the issue of full infrastructure resolution. Resolving dependencies
```

Comment:
I'd re-punctuate this: 

Batali aims to solve the issue of full infrastructure resolution: resolving all dependencies in an infrastructure repository. Resolving a single dependency...